src/common/cluster/cluster.ts

```diff
@@ -656,7 +655,7 @@ export class Cluster implements ClusterModel {
       const namespaceList = [ctx?.namespace].filter(isDefined);
 
       if (namespaceList.length === 0 && error instanceof HttpError && error.statusCode === 403) {
-        const { response } = error as HttpError & { response: Response };
+        const { response } = error as HttpError & { response: { body: unknown }};
```

Comment:
It is still using the `request` package because that is what the `@kubernetes/client` package uses until https://github.com/kubernetes-client/javascript/issues/754 is resolved.